python_benchmarks/qr_codes/qr.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python
+
+"""qr.py
+Usage example:
+python qr.py -o out.yaml -p qrcodes/detection
+-H, --help - show help
+-o, --output - output file (default out.yaml)
+-p, --path - input dataset path (default qrcodes/detection)
+-a, --accuracy - input accuracy (default 47)
```

Comment:
what is accuracy and what 47 means?

Reply:
fixed

Reply:
It's still there.

Reply:
set default `accuracy` as 20 pixels. This value for this dataset (with large images) is a good marker.